shared/test/test_table_limits.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'test_helper'
+require 'fake_redis'
```

Comment:
Changed to the canonical fakeredis.  (I checked and the gem actually defines both fake_redis and fakeredis)

![image](https://cloud.githubusercontent.com/assets/5198586/13791861/49aba348-eaac-11e5-8e66-8635c9132937.png)